.github/CODEOWNERS

```diff
@@ -935,9 +935,8 @@
 /.github/workflows/                                           @Azure/azure-sdk-eng
 /.config/1espt/                                               @benbp @weshaggard
 
-# Removing owners for these files to not require codeowner approval for changes to them given they are shared with project
+# Removing owners for this file to not require codeowner approval for changes to them given they are shared with project
 /eng/versioning/version_client.txt
-/eng/versioning/version_data.txt
```

Comment:
Do we think we will need a v2 client when we start getting into that work or will it follow all the client rules?